src/LanguageServer/Test/GoToDefinitionTests.cs

```diff
@@ -204,6 +204,46 @@ class D(C):
             reference.range.Should().Be(2, 4, 2, 5);
         }
 
+        [TestMethod, Priority(0)]
+        public async Task MultipleInheritanceSuperShouldWalkDecendants() {
+            var testModPath = TestData.GetTestSpecificUri("test.py");
+            const string code = @"
+        class GrandParent:
+            def dowork(self):
+                return 1
+
+        class Dad(GrandParent):
+            def dowork(self):
+                return super().dowork()
+
+        class Mom():
+            def dowork(self):
+                return 2
+
+        class Child(Dad, Mom):
+            def child_func(self):
+                x = super()
```

Comment:
`x` is a local variable and would not be persisted.

Reply:
I can remove this line

Reply:
No big deal, misread the test class.